Source/Scene/TileOrientedBoundingBox.js

```diff
@@ -0,0 +1,65 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/DeveloperError',
+        '../Core/OrientedBoundingBox'
+    ], function(
+        defaultValue,
+        defined,
+        defineProperties,
+        DeveloperError,
+        OrientedBoundingBox) {
+    "use strict";
+
+    var TileOrientedBoundingBox = function(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+        this.orientedBoundingBox = OrientedBoundingBox.fromRectangle(options.rectangle, options.minimumHeight, options.maximumHeight);
+    };
+
+    defineProperties(TileOrientedBoundingBox.prototype, {
+        /**
+         * The underlying bounding volume
+         *
+         * @memberof TileOrientedBoundingBox.prototype
+         *
+         * @type {Object}
+         * @readonly
+         */
+        boundingVolume : {
+            get : function() {
+                return this.orientedBoundingBox;
+            }
+        }
+    });
+
+    /**
+     * Computes the distance between this bounding box and the camera attached to frameState.
+     * @param {FrameState} [frameState] The frameState to which the camera is attached.
+     * @returns {Number} The distance between the camera and the bounding box in meters. Returns 0 if the camera is inside the bounding volume.
+     */
+    TileOrientedBoundingBox.prototype.distanceToCamera = function(frameState) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(frameState)) {
+            throw new DeveloperError('frameState is required.');
+        }
+        //>>includeEnd('debug');
+        return Math.sqrt(this.orientedBoundingBox.distanceSquaredTo(frameState.camera.positionWC));
```

Comment:
@e-andersson did you test `distanceSquaredTo` to see if the numbers were reasonable?  There are unit tests for this, but it is not used anywhere else in Cesium so I just want to be sure.